recipe/meta.yaml

```diff
@@ -27,15 +27,18 @@ source:
     - nvcc.profile.patch.win  # [win]
 
 build:
-  number: 2
+  number: 3
   skip: true  # [osx or win]
 
 outputs:
   - name: cuda-nvcc
     files:
-      - bin                                               # [linux]
-      - lib/libnvptxcompiler_static.a                     # [linux]
-      - nvvm                                              # [linux]
+      - targets/{{ target_name }}/bin/nvcc  # [linux]
+      - bin                                 # [linux]
+      - lib/libnvptxcompiler_static.a       # [linux]
+      - nvvm                                # [linux]
+      - etc/conda/activate.d/~cuda-nvcc_activate.sh      # [linux]
+      - etc/conda/deactivate.d/~cuda-nvcc_deactivate.sh  # [linux]
```

Comment:
We need these somewhere so I put them here for now. If there is a better home for them we can move them as needed.

Reply:
Wouldn't we want these still in `cuda-nvcc_{{ target_platform }}`?